src/tests/Microsoft.PowerFx.Interpreter.Tests/FileExpressionEvaluationTests.cs

```diff
@@ -92,7 +92,7 @@ public void RunMutationTests(string file)
 
             if (result.Fail > 0)
             {
-                Assert.Equal(string.Empty, result.Output);
+                Assert.True(false, result.Output);
```

Comment:
When fail, a test case was showing a partial error message, making it difficult to understand what was broken. This fix shows the entire error message.